memory/libhugetlbfs.py

```diff
@@ -1,5 +1,23 @@
 #!/usr/bin/env python
 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Santhosh G <santhog4@linux.vnet.ibm.com>
+#
+# Based on code by aganti@google.com (Ashwin Ganti)
+# copyright : 2008 Google, Inc.
+# https://github.com/autotest/autotest-client-tests/tree/master/libhugetlbfs
```

Comment:
OK I did the homework and the original author I found was: `Yao Fei Zhu <walkinair@cn.ibm.com>` in `2006`, commit id `9e25240f3a1ba489b144c116ec599ea54ad4a0b5`. Then it was moved to `client/tests/libhugetlbfs` and `control` file with license was added by `John Admanski<jadmanski@google.com>` in commit `95b58c8931e61e9402b8d63c6ad83a6acd7e1502` telling the author is `aganti@google.com (Ashwin Ganti)`, but I don't see any contributions from him. There are many contributions by @lmr and @cevich so maybe we can ask them for opinion.

My opinion would be that the original author is the first committer in 2006. The author from `control` file did not contribute any code, so I'd ignore him, unless he claims and proves the contribution. As for the rest of the contributors, they did not claim the authorship, so I'd silently ignore them, unless they claim the authorship. Then we should include them.

PS: This is exactly why GPL mandates to list the authors at the beginning of each file, to avoid such detective work...